x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/GeoPointFieldScript.java

```diff
@@ -49,20 +51,13 @@
         lookup,
         ctx
     ) {
+        private final GeoPoint scratch = new GeoPoint();
+
```

Comment:
I have another question, is that ok? not sure if execute method needs to be thread safe.

Reply:
this looks ok, we create a new instance for every query.

Reply:
yea we are good, sorry I forgot to answer earlier.